servers/mu/src/domain/utils.test.js

```diff
@@ -84,4 +84,35 @@ describe('utils', () => {
       )
     })
   })
+
+  describe('swallow', () => {
+    test('should resolve the original promise', async () => {
+      const safeFunction = swallow(() => Promise.resolve('hello world'))
+      const result = await safeFunction()
+
+      assert.strictEqual(result, 'hello world')
+    })
+
+    test('should resolve undefined for a rejected promise', async () => {
+      const safeFunction = swallow(() => Promise.reject(new Error('This error should be swallowed')))
+      const result = await safeFunction()
+
+      assert.strictEqual(result, undefined)
+    })
+
+    test('should return the result of the function', () => {
+      const safeFunction = swallow(() => 'hello world')
+
+      const result = safeFunction()
+      assert.equal(result, 'hello world')
+    })
+
+    test('should return undefined for a thrown error', () => {
+      const safeFunction = swallow(() => { throw new Error('This error should be swallowed') })
+
+      const result = safeFunction()
+
+      assert.strictEqual(result, undefined)
+    })
```

Comment:
Could we make the spacing between blocks consistent across these tests? Doesn't matter which, just as long as it's consistent.